content/docs/authorization/deployment/rpm/_index.md

```diff
@@ -35,6 +35,12 @@ yum install -y container-selinux
 
 yum install -y https://rpm.rancher.io/k3s/stable/common/centos/8/noarch/k3s-selinux-0.4-1.el8.noarch.rpm
 
+### Dark Sites
+
+For environments where `yum` will not work, obtain the supported version of container-selinux for your OS version and install it.
```

Comment:
Is there a landing page link we can add here for the download?

Reply:
Sort of. There is https://github.com/containers/container-selinux/releases and https://pkgs.org/download/container-selinux. The GitHub releases page isn't clear about supported versions but the second link is. Should we use the second link as an example landing page?

Reply:
Updated to use https://centos.pkgs.org as an example to download from.